picard/script.py

```diff
@@ -1258,8 +1258,10 @@ def func_datetime(parser, format=None):
     # Handle case where format evaluates to ''
     if not format:
         format = '%Y-%m-%d %H:%M:%S'
-
-    return datetime.datetime.now(tz=local_tz).strftime(format)
+    try:
+        return datetime.datetime.now(tz=local_tz).strftime(format)
+    except ValueError:
+        return 'Invalid Format'
```

Comment:
It looks like all the other script functions return a blank string on error or a default value which makes sense. Invalid Format seems to be neither of those. Should we just raise the error here or return a blank string instead?

Reply:
Updated to return a blank string to be consistent with exception returns from other functions.